src/commands/createNewProject/CSharpScriptProjectCreator.ts

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { localize } from "../../localize";
+import { ProjectRuntime, TemplateFilter } from "../../ProjectSettings";
+import { funcHostTaskId, funcHostTaskLabel } from "./IProjectCreator";
+import { ScriptProjectCreatorBase } from './ScriptProjectCreatorBase';
+
+export class CSharpScriptProjectCreator extends ScriptProjectCreatorBase {
+    public readonly templateFilter: TemplateFilter = TemplateFilter.Verified;
+    public readonly runtime: ProjectRuntime = ProjectRuntime.beta;
+
+    public getTasksJson(): {} {
+        return {
+            version: '2.0.0',
+            tasks: [
+                {
+                    label: funcHostTaskLabel,
+                    identifier: funcHostTaskId,
+                    type: 'shell',
+                    command: 'func host start',
+                    isBackground: true,
+                    presentation: {
+                        reveal: 'always'
+                    }
+                }
+            ]
+        };
+    }
+
+    public getLaunchJson(): {} {
+        return {
+            version: '0.2.0',
+            configurations: [
+                {
+                    name: localize('azFunc.attachToNetCoreFunc', "Attach to .NET Core Functions"),
+                    type: 'coreclr',
+                    request: 'attach',
```

Comment:
I'm probably doing something incorrectly, butt seems like C# Scripts attach, but I can't set breakpoints.

Reply:
It was working for me. I'll look into separately
https://github.com/Microsoft/vscode-azurefunctions/issues/180